packages/ckeditor5-image/tests/imageinsert/ui/imageinsertpanelview.js

```diff
@@ -294,5 +294,32 @@ describe( 'ImageUploadPanelView', () => {
 
 			expect( view.imageURLInputValue ).to.equal( 'xyz' );
 		} );
+
+		it( 'should trim input value', () => {
+			const form = view.getIntegration( 'insertImageViaUrl' );
+
+			form.fieldView.element.value = '   ';
+			form.fieldView.fire( 'input' );
+
+			expect( view.imageURLInputValue ).to.equal( '' );
+
+			form.fieldView.element.value = '   test   ';
+			form.fieldView.fire( 'input' );
+
+			expect( view.imageURLInputValue ).to.equal( 'test' );
+		} );
+
+		it( 'binds saveButtonView#isEnabled to URL input value', () => {
+			const form = view.getIntegration( 'insertImageViaUrl' );
+			const saveButtonView = view.template.children[ 1 ].children.first;
+
+			expect( !!saveButtonView.isEnabled ).to.be.false;
```

Comment:
```suggestion
			expect( saveButtonView.isEnabled ).to.be.false;
```

I'm adding suggestions to this - as I think that we should have consistent values for `isFooBar` properties. We document this as `Boolean` so I think that this should be done on the bounding level.